chef/templates/default/manifest.xml.erb

```diff
@@ -15,7 +15,7 @@
             <service_fmri value="svc:/milestone/multi-user"/>
         </dependency>
         <exec_method timeout_seconds="60" type="method" name="start"
-            exec="<%= @command.encode(:xml => :text) %>">
+            exec="<%= "nohup #{@command} &".encode(:xml => :text) %>">
```

Comment:
Why are we now going to background this process? Just need some context so that I know how to implement in the poise-service-aix provider.